homeassistant/components/skybell/switch.py

```diff
@@ -44,9 +58,8 @@ def __init__(self, device, switch_type):
         """Initialize a light for a Skybell device."""
         super().__init__(device)
         self._switch_type = switch_type
-        self._name = "{} {}".format(
-            self._device.name, SWITCH_TYPES[self._switch_type][0]
-        )
+        metadata = SWITCH_TYPES[self._switch_type]
+        self._name = f"{self._device.name} {metadata.name}"
```

Comment:
```suggestion
        self._attr_name = f"{self._device.name} {metadata.name}"
```
With that the `name` property could be remove

Reply:
Done, but very soon we have rewritten the integration.